stdlib/ballerina-builtin/src/main/ballerina/util/natives.bal

```diff
@@ -24,19 +25,35 @@ public struct Locale {
     string countryCode;
 }
 
+@Description {value:"Represent errors related to all base64 encoders and decoders"}
+@Field {value:"message: The error message"}
+@Field {value:"cause: The error which caused the entity error"}
+public struct Base64Error {
```

Comment:
`Base64Error` doesn't sound right. Can't we just have two errors structs for encoding and decoding?